specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json

```diff
@@ -1654,6 +1654,20 @@
       },
       "x-ms-discriminator-value": "Microsoft.Compute/availabilitySets"
     },
+    "SubnetsResourceSettings": {
```

Comment:
For semantic considerations, could we rename this to `SubnetResourceSetting` if possible?

Reply:
taken.